spec/router_spec.lua

```diff
@@ -162,6 +162,34 @@ describe("Router", function()
           -- path /bb/var1/cc/dd overlap with handle 4
           assert.equal("5", router:match("/bb/var1/cc/dd"))
         end)
+
+        it("trailing variable with trailing slash off", function()
+          local router = Router.new({
+            {
+              paths = { "/aa/{var1}" },
+              handler = "1"
+            },
+            {
+              paths = { "/aa/{var1}/bb" },
+              handler = "2",
+            }
+          })
+
+          assert.equal("1", router:match("/aa/var1"))
+          assert.equal("2", router:match("/aa/var1/bb"))
+        end)
+
+        it("trailing variable with trailing slash on", function()
+          local router = Router.new({
+              {paths = {"/aa/{id}"}, handler="1"},
+              {paths = {"/aa/{id}/bb"}, handler="2"},
+          }, {
+              trailing_slash_match = true
+          })
+
+          assert.equal("1", router:match("/aa/var1"))
+          assert.equal("2", router:match("/aa/var1/bb"))
+        end)
```

Comment:
I prefer to add more routes in here rather than adding a new `it`

https://github.com/vm-001/lua-radix-router/blob/8fbc2f162963322d9959b820f022a085ae8a11b9/spec/router_spec.lua#L622